xsl.json

```diff
@@ -0,0 +1,38 @@
+{
+    "tm|tmatch": "xsl:template[match mode]",
+    "tn|tname": "xsl:template[name]",
+    "call": "xsl:call-template[name]",
+    "ap": "xsl:apply-templates[select mode]",
+    "api": "xsl:apply-imports",
+    "imp": "xsl:import[href]",
+    "inc": "xsl:include[href]",
+    "ch": "xsl:choose",
+    "wh|xsl:when": "xsl:when[test]",
+    "ot": "xsl:otherwise",
+    "if": "xsl:if[test]",
+    "par": "xsl:param[name]",
+    "pare": "xsl:param[name select]",
+    "var": "xsl:variable[name]",
+    "vare": "xsl:variable[name select]",
+    "wp": "xsl:with-param[name select]",
+    "key": "xsl:key[name match use]",
+    "elem": "xsl:element[name]",
+    "attr": "xsl:attribute[name]",
+    "attrs": "xsl:attribute-set[name]",
+    "cp": "xsl:copy[select]",
+    "co": "xsl:copy-of[select]",
+    "val": "xsl:value-of[select]",
+    "for|each": "xsl:for-each[select]",
+    "tex": "xsl:text",
+    "com": "xsl:comment",
+    "msg": "xsl:message[terminate=no]",
+    "fall": "xsl:fallback",
+    "num": "xsl:number[value]",
+    "nam": "namespace-alias[stylesheet-prefix result-prefix]",
+    "pres": "xsl:preserve-space[elements]",
+    "strip": "xsl:strip-space[elements]",
+    "proc": "xsl:processing-instruction[name]",
+    "sort": "xsl:sort[select order]",
+    "choose+": "xsl:choose>xsl:when+xsl:otherwise",
```

Comment:
> And so I propose to look for * or ) or end of stream. Because its exactly in these 3 cases that there is no sibling that is following the +

In current Emmet implementation it’s possible to use such abbreviations in the middle: `ul++div`.